### ISSUE TITLE
Accessibility: Focus indicator missing

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51894]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The button to dismiss the error banner does not have a visible outline on focus. There are some CSS styles targeting this element with outline:none !important; that many be conflicting with focus outlines.

![Image](https://github.com/user-attachments/assets/e5944b9e-2965-4506-9113-48fe6e5d2dca)

### Component or Page: 

Page: Cart (Modern) 

### Name affected element:

Dismiss this notice button

### Relevant code snippets:

N/A

### Recommended fix:

Modify the CSS in your stylesheet to ensure the element has a 2px solid outline with a 2px outline offset on focus.
